src/core/Timer.hpp

```diff
@@ -19,5 +20,6 @@ class CTimer {
     std::function<void(std::shared_ptr<CTimer> self, void* data)> cb;
     void*                                                         data = nullptr;
     std::chrono::system_clock::time_point                         expires;
-    bool                                                          wasCancelled = false;
-};
\ No newline at end of file
+    bool                                                          wasCancelled     = false;
+    bool                                                          allowForceUpdate = false;
```

Comment:
or should I rename here? not to be the same as in [IWidget.hpp](https://github.com/hyprwm/hyprlock/pull/169/files#diff-6cf15676b59d5669af3c5248d657b158e129f110267c03965273f6d713b1a917R21)?

Reply:
it fine